mico-core/src/main/java/io/github/ust/mico/core/web/ApplicationController.java

```diff
@@ -184,46 +208,62 @@
     public ResponseEntity<Resources<Resource<MicoService>>> getServicesFromApplication(@PathVariable(PATH_VARIABLE_SHORT_NAME) String shortName,
                                                                                        @PathVariable(PATH_VARIABLE_VERSION) String version) {
         MicoApplication micoApplication = getApplicationFromDatabase(shortName, version);
-        List<MicoService> micoServices = micoApplication.getServices();
+        List<MicoService> micoServices = serviceRepository.findAllByApplication(micoApplication.getShortName(), micoApplication.getVersion());
         List<Resource<MicoService>> micoServicesWithLinks = ServiceController.getServiceResourcesList(micoServices);
         return ResponseEntity.ok(
             new Resources<>(micoServicesWithLinks,
                 linkTo(methodOn(ApplicationController.class).getServicesFromApplication(shortName, version)).withSelfRel()));
     }
 
+    // TODO: Deployment info necessary to add new service to application!
```

Comment:
Is there another issue opened for that? Or should it be solved in this PR?

Reply:
I can do that within this pull request. Do we assume, that the corresponding application in the database already has the deployment information for the service to add? Or should we apply the default service deployment information so that it can be modified later on through another endpoint?

Reply:
See #476.